<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreUI</class>
 <widget class="QMainWindow" name="CoreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸识别</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>471</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="title">
     <string>日志</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>461</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="systemSettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="faceRecognizerCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>数据库未完成初始化，请检查</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(90, 210, 250);
color: #fefefe;

padding-left:10px;
font: &quot;Microsoft YaHei UI&quot;;


</string>
     </property>
     <property name="text">
      <string>人脸识别</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="faceTrackerCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
background-color: rgb(90, 210, 250);
color: #fefefe;
font: &quot;Microsoft YaHei UI&quot;;
padding-left:10px

</string>
     </property>
     <property name="text">
      <string>人脸跟踪</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="realTimeCaptureLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>551</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Microsoft JhengHei&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ff0000;&quot;&gt;请开启摄像头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(90, 210, 250);
color: #fefefe;
font: &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startWebcamButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(90, 210, 250);
color: #fefefe;
font: &quot;Microsoft YaHei UI&quot;;
</string>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openRecordButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>670</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(90, 210, 250);
color: #fefefe;
font: &quot;Microsoft YaHei UI&quot;;
</string>
    </property>
    <property name="text">
     <string>增加用户</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(90, 210, 250);
color: #fefefe;
font: &quot;Microsoft YaHei UI&quot;;
</string>
    </property>
    <property name="text">
     <string>管理信息</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
